README.md

```diff
@@ -1,32 +1,30 @@
-Polysh
-~~~~~~
+[![Build Status](https://travis-ci.org/innogames/polysh.svg?branch=master)](https://travis-ci.org/innogames/polysh)
+
+# Polysh
 
 Polysh (formerly called Group Shell or gsh) is a remote shell multiplexor. It
 lets you control many remote shells at once in a single shell. Unlike other
 commands dispatchers, it is interactive, so shells spawned on the remote hosts
 are persistent. It requires only a SSH server on the remote hosts, or some other
 way to open a remote shell.
 
-Requirements
-~~~~~~~~~~~~
+# Requirements
 
-Python >= 2.4 is required.
+Python >= 3.something is required.
```

Comment:
So we are not adding Python 3 support, but making it Python 3 only.  I am fine with that.  We shouldn't claim it would work on any Python 3 version, though.  It would most likely not.  Requiring minimum Python 3.4 or 3.5 would be reasonable.  We can adjust the tests to check the versions we support.

Reply:
Python 3.2 support might of interest to some, but it seems I broke 3.2 support already[1] and it's probably not important for us. Python 3.3 works, but AFAIK it's pretty irrelevant. Does any big LTS distro ship this? 3.4 sounds like a reasonable minimum version to me then.

[1] https://travis-ci.org/innogames/polysh/jobs/328441318

Reply:
Yeah, let's go for 3.4.

Reply:
Done